src/dotenv_vault/__version__.py

```diff
@@ -1,7 +1,7 @@
 __title__ = "python-dotenv-vault"
 __description__ = "Decrypt .env.vault file."
 __url__ = "https://github.com/dotenv-org/python-dotenv-vault"
-__version__ = "0.6.0"
```

Comment:
this is a new feature rather than a patch. bump to `0.7.0`

Reply:
done